core/templates/pages/contributor-dashboard-page/opportunities-list-item/opportunities-list-item.component.spec.ts

```diff
@@ -203,4 +203,33 @@ describe('Opportunities List Item Component', () => {
         expect(component.correspondingOpportunityDeleted).toBeFalse();
       });
   });
+
+  describe('when translation card opportunity is provided', () => {
+    beforeEach(() => {
+      component.opportunity = {
+        id: '1',
+        labelText: 'Label text',
+        labelColor: '#fff',
+        progressPercentage: 50,
+        inReviewCount: 20,
+        totalCount: 50,
+        translationsCount: 25,
+        translationLengthInWords: 30
+      };
+      component.opportunityType = 'translation';
+      component.clickActionButton.emit =
+        () => jasmine.createSpy('click', () => {});
+      component.labelRequired = true;
+      component.opportunityHeadingTruncationLength = 35;
+      fixture.detectChanges();
+      component.ngOnInit();
+    });
+
+    it('should show translation size label for translation suggestions', () => {
```

Comment:
We should add a test for the `short` case.

Reply:
I dont think we need to have test cases now, as we have shifted the logic from component to html file.